pynest/examples/evaluate_quantal_stp_synapse.py

```diff
@@ -69,124 +69,156 @@
        http://dx.doi.org/10.1016/S0893-6080(01)00144-7
 
 """
+
 import nest
-import nest.voltage_trace
 import numpy
 import matplotlib.pyplot as plt
 
-nest.ResetKernel()
-
 ################################################################################
 # On average, the ``quantal_stp_synapse`` converges to the ``tsodyks2_synapse``,
 # so we can compare the two by running multiple trials.
 #
-# First we define the number of trials as well as the number of release sites.
+# First we define simulation time step and random seed
+
+resolution = 0.1  # [ms]
+seed = 12345
+
+# We define the number of trials as well as the number of release sites.
 
 n_syn = 10.0  # number of synapses in a connection
 n_trials = 100  # number of measurement trials
 
+# The pre-synaptic neuron is driven by an injected current for a part of each
+# simulation cycle. We define here the parameters for this stimulation cycle.
+
+I_stim = 376.0   # [pA] stimulation current
+T_on = 500.0     # [ms] stimulation is on
+T_off = 1000.0   # [ms] stimulation is off
+
+T_cycle = T_on + T_off   # total duration of each stimulation cycle
+
 ###############################################################################
-# Next, we define parameter sets for facilitation
+# Next, we define parameter sets for facilitation and initial weight.
 
-fac_params = {"U": 0.02, "u": 0.02, "tau_fac": 500.,
-              "tau_rec": 200., "weight": 1.}
+fac_params = {"U": 0.02,
+              "u": 0.02,
+              "tau_fac": 500.,
+              "tau_rec": 200.,
+              "weight": 1.}
 
 ###############################################################################
 # Then, we assign the parameter set to the synapse models
 
-tsodyks_params = dict(fac_params, synapse_model="tsodyks2_synapse")  # for tsodyks2_synapse
-quantal_params = fac_params.copy()  # for quantal_stp_synapse
+tsyn_params = fac_params  # for tsodyks2_synapse
+qsyn_params = tsyn_params.copy()  # for quantal_stp_synapse
 
-tsodyks_params['x'] = tsodyks_params['U']
-quantal_params['n'] = n_syn
+tsyn_params["x"] = tsyn_params["U"]
+qsyn_params["n"] = n_syn
 
 ###############################################################################
 # To make the responses comparable, we have to scale the weight by the
 # number of synapses.
```

Comment:
change `number of synapses` to `number of vesicles`

Reply:
I used "release sites" now, since that term is also used in the comment at the top of the file.

Reply:
yes, sorry I meant `number of release sites`